Exercise_D.js

```diff
@@ -15,6 +15,14 @@ Use a combination of conditionals and loops to create your answer for this quest
 
 function multiplyNumber(number) {
   // CODE HERE
+  if (number < 0) return "ERROR";
+  else if (number >= 1000000) return number;
+  else
+    while (number <= 1000000) {
+      number = number * 10;
+      // console.log(number);
+    }
+  return number;
 }
 
-console.log(multiplyNumber(10));
+console.log(multiplyNumber(22));

```

Comment:
For this exercise, the answer that you have given does not check for when the number is 0 or when it is not a number. It has only checked when a number is smaller/bigger than or equal to 100000. We would have to add additional conditionals to this statement. 
You can use isNaN to check if its a number and or also use typeof